### PR TITLE
Pricing restaurants richard

### PR DESCRIPTION
**Pull request by richardcspds**
Working Master template and R2-R6 Scenarios.

### CODE REVIEW

File: pages/BusinessPage.js [hunk 1]
```diff
@@ -19,8 +19,11 @@ class BusinessPage extends BasePage {
             //labels
         this.bpBusinessNameLabel = "//h1/span[contains(text(), 'the name')]";
         this.bpAddressConfirmationLabel = "//h1[text()= 'Is this your business?']";
+        this.bpPolicyStartLabel = "//h1[text()= 'When do you want your policy to begin?']";
         this.bpEmployeesQuestionLabel = "//div[contains(text(), 'How many of your employees work more than 20 hours a week?')]"
-        this.bpXmodQuestionLabel = "//div[contains(text(), 'Do you have an experience modification factor (xMod)?')]"
```
Comment (benlizar) [inline pages/BusinessPage.js:23]: no more xMod question?

File: pages/EstimatePage.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { element } from "../utils/Decorators"
+
+class EstimatePage{
```
Comment (benlizar) [inline pages/EstimatePage.js]: Missing the "BasePase" exntesion

File: pages/fragments/sliderFragments.js [hunk 3]
```diff
@@ -19,11 +19,15 @@ class SliderFragment {
         var interval = sliderComponent[4].value
```
Comment (benlizar) [inline pages/fragments/sliderFragments.js]: Unused variable, can we delete it?

**Review by benlizar — CHANGES_REQUESTED** [review]

File: pages/BusinessPage.js (hunk 1 above)
Comment (richardcspds) [reply pages/BusinessPage.js:23]: Changed the name of the variable as per its HTML tag "div" as it is not a label

File: pages/EstimatePage.js (hunk 2 above)
Comment (richardcspds) [reply pages/EstimatePage.js]: Will add

File: pages/fragments/sliderFragments.js (hunk 3 above)
Comment (richardcspds) [reply pages/fragments/sliderFragments.js]: Sure, deleting.

**Review by benlizar — APPROVED** [review]

**Pull request merged**